frontend/src/scenes/Home/Login.tsx

```diff
@@ -112,6 +125,16 @@ export default function Login({ locale }: SceneProps) {
         >
           {STRINGS[locale].FACEBOOK_SIGN_UP}
         </FacebookButton>
+        <SupportButton
+          className="sign-up-button"
+          disableElevation
+          onClick={() => {
+            (window as any).location = 'https://m.me/callhomesg';
```

Comment:
`window.location =` shouldn't be used for plain redirects. `window.location =` is a programmatic redirect and should be reserved for things that are not repeatable.

For a plain redirect, use an `<a>` tag instead.